app/controllers/records_controller.rb

```diff
@@ -0,0 +1,58 @@
+class RecordsController < ApplicationController
+  before_action :authenticate_user!
+
+  def index
+    @records = current_user.records.all
+  end
+
+  def show
+    render :show, locals: { record: record }
```

Comment:
We need locals? We use in show the local `record` but we have `@record` too

```suggestion
```

Reply:
I think the use of variable instances inside views was a smell. But I'm okay with your suggestion